.github/workflows/check.yml

```diff
@@ -63,26 +63,13 @@ jobs:
                   components: rust-src, rustfmt, clippy
                   default: true
 
-            - name: Set up Rust nightly
-              uses: actions-rs/toolchain@v1
-              with:
-                  profile: minimal
-                  toolchain: nightly
-
             - name: Install evcxr
               if: matrix.os != 'windows-latest' && matrix.rust-version >= '1.40.0' # BACKCOMPAT: Rust 1.39.0
               uses: actions-rs/cargo@v1
               with:
                   command: install
                   args: evcxr_repl
 
-            - name: Install grcov
```

Comment:
I know that we don't run coverage tests on CI right now and these steps are useless currently. Are you going to return these steps when coverage tests will be more stable?

Reply:
Yes.